openrr-planner/Cargo.toml

```diff
@@ -31,3 +31,5 @@ kiss3d = "0.30"
 structopt = "0.3"
 tracing-subscriber = "0.2"
 urdf-viz = "0.24"
+
+approx = "0.4.0"
```

Comment:
And could you move this line to L30?
We are sorting lines alphabetically.

I mean,

```toml
[dev-dependencies]
approx = "0.4.0"
kiss3d = "0.30"
...
```

Reply:
The example is Useful for me.

Thank you